src/commands/mod.rs

```diff
@@ -2298,6 +2376,294 @@ fn cmd_new(ui: &mut Ui, command: &CommandHelper, args: &NewArgs) -> Result<(), C
     Ok(())
 }
 
+fn cmd_next(ui: &mut Ui, command: &CommandHelper, args: &NextArgs) -> Result<(), CommandError> {
+    let mut workspace_command = command.workspace_helper(ui)?;
+    let edit = args.edit;
+    let amount = args.amount;
+    assert!(amount >= 1);
+    let current_wc_id = workspace_command
+        .get_wc_commit_id()
+        .ok_or_else(|| user_error("This command requires a working copy"))?;
+    let current_wc = workspace_command.repo().store().get_commit(current_wc_id)?;
+    // TODO(#NNN): This currently depends on order in which the parents are
+    // returned, make it configurable.
+    let parents = current_wc.parents();
+    let parent = parents
+        .first()
+        .ok_or_else(|| user_error("unable to determine parent"))?;
+    let parent_commit = RevsetExpression::commit(parent.id().clone());
+    // Collect all descendants of our parent.
+    // TODO(#NNN) We currently cannot deal with multiple children, which result
+    // from branches. Fix it when --interactive is implemented.
+    let mut children_revset = RevsetExpression::children(&parent_commit);
+    // let children: Vec<Commit> = RevsetExpression::descendants(&parent_commit)
+    //     .resolve(workspace_command.repo().as_ref())?
+    //     .evaluate(workspace_command.repo().as_ref())?
+    //     .iter()
+    //     .commits(workspace_command.repo().store())
+    //     .try_collect()?;
+
+    let current_id = current_wc.id();
```

Comment:
nit: this seems to be used only on the next line, and it's very simple, so inline it?

Reply:
I've inlined all cases.